openbb_platform/core/openbb_core/app/provider_interface.py

```diff
@@ -253,14 +253,22 @@ def _create_field(
         annotation = field.annotation
 
         additional_description = ""
+        choices: Dict = {}
         if extra := field.json_schema_extra:
-            providers = []
-            for p, v in extra.items():  # type: ignore[union-attr]
+            providers: List = []
+            for p, v in extra.items():  # type: ignore
                 if isinstance(v, dict) and v.get("multiple_items_allowed"):
                     providers.append(p)
+                    choices[p] = {"multiple_items_allowed": True, "choices": v.get("choices")}  # type: ignore
```

Comment:
why are we assuming the `multiple_items_allowed` will be true here? shouldn't it be getting the value as it was before? `v.get("multiple_items_allowed")`

Reply:
It's not an assumption, the `if` statement evaluates `v["multiple_items_allowed"]` for truthiness.